### PR TITLE
Format code and update manifests

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: Ben Ye <yb532204897@gmail.com>

### What type of PR is this? 
_(bug/feature/cleanup/docs/design/test/chore/refactor..)_

### What this PR does / why we need it:

Format code and update manifests based on API changes.

### Which issue(s) this PR fixes(can be a reference to existing issue or jira/bz):

_Fixes #_

### Pre-checks:
- [ ] manually tested latest changes against crc/k8s
- [ ] run the `make coverage` command to generate new calculated coverage

### CODE REVIEW

File: deploy/crds/gcp_v1alpha1_projectclaim_crd.yaml [hunk 1]
```diff
@@ -46,6 +46,20 @@ spec:
...
+              type: boolean
+            ccsProjectID:
+              type: string
+            ccsSecretRef:
+              properties:
+                name:
+                  type: string
+                namespace:
+                  type: string
+              required:
```
Comment (rafael-azevedo) [inline deploy/crds/gcp_v1alpha1_projectclaim_crd.yaml:59]: remove required field here

File: deploy/crds/gcp_v1alpha1_projectreference_crd.yaml [hunk 2]
```diff
@@ -46,6 +46,18 @@ spec:
...
           properties:
+            ccs:
+              type: boolean
+            ccsSecretRef:
+              properties:
+                name:
+                  type: string
+                namespace:
+                  type: string
+              required:
```
Comment (rafael-azevedo) [inline deploy/crds/gcp_v1alpha1_projectreference_crd.yaml:57]: remove required field here

File: deploy/crds/gcp_v1alpha1_projectreference_crd.yaml [hunk 3]
```diff
@@ -107,11 +119,11 @@ spec:
               type: array
             state:
               type: string
+          required:
```
Comment (rafael-azevedo) [inline deploy/crds/gcp_v1alpha1_projectreference_crd.yaml:122]: remove required field here

File: deploy/crds/gcp_v1alpha1_projectreference_crd.yaml [hunk 4]
```diff
@@ -107,11 +119,11 @@ spec:
...
+          - conditions
           type: object
-#      type: object # This line causes the CRD to stop working on openshift (last checked on 3.11.154)
-                    # The reason for that is the v1beta1 CRD api changed upstread and the tool
-                    # which generates this file `operator-sdk` uses the upstream api.
-                    # see issue https://github.com/jetstack/cert-manager/issues/2200
+      required:
```
Comment (rafael-azevedo) [inline deploy/crds/gcp_v1alpha1_projectreference_crd.yaml:125]: remove required field here

**Review by rafael-azevedo — CHANGES_REQUESTED** [review]

File: deploy/crds/gcp_v1alpha1_projectclaim_crd.yaml (hunk 1 above)
Comment (yeya24) [reply deploy/crds/gcp_v1alpha1_projectclaim_crd.yaml:59]: It is needed.

File: deploy/crds/gcp_v1alpha1_projectreference_crd.yaml (hunk 2 above)
Comment (yeya24) [reply deploy/crds/gcp_v1alpha1_projectreference_crd.yaml:57]: It is needed.

**Comment by RiRa12621** [comment]
@yeya24 can you please fill out the template, so it's a bit more obvious what's going to happen in this PR before looking at the code itself?

**Comment by yeya24** [comment]
@RiRa12621 Sorry for the inconvenience, I just updated the description. Thanks!

**Review by rafael-azevedo — APPROVED** [review]

**Pull request merged**